ui/src/containers/AppsTableContainer.js

```diff
@@ -30,7 +30,13 @@ export class AppsTableContainer extends React.Component {
   getTable () {
     return (
       <div className="apps-table">
-        <AppsTable columns={this.props.columns} rows={this.props.apps.rows} sortBy={this.sortBy} onSort= {this.onSort} onRowClick={this.onRowClick}/>
+        <AppsTable
+          columns={this.props.columns}
+          rows={this.props.apps.rows}
+          sortBy={this.props.sortBy}
+          onSort={this.onSort}
```

Comment:
`this.props.onSort?`

Reply:
No, as `onSort()` is defined as a handler in the file, as opposed to coming in via a prop